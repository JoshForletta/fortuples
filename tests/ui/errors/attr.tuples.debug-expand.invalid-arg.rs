use fortuples::fortuples;

struct Container<T>(T);

fortuples! {
    #[tuples::debug_expand(invalid = "")]
    impl Container<#Tuple> {}
}

fn main() {}
