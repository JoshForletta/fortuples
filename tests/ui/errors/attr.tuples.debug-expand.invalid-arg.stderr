error: `path` is expected
 --> tests/ui/errors/attr.tuples.debug-expand.invalid-arg.rs:6:28
  |
6 |     #[tuples::debug_expand(invalid = "")]
  |                            ^^^^^^^
